app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -112,9 +112,9 @@
   <!-- monthly hypertension control view -->
   <div class="card">
     <a href="#"  onclick="openWindow('progress-help-controlled', 'progress-start'); return false" class="info"><span>?</span></a>
-    <h3><%= t("analytics.hypertension_controlled") %></h3>
+    <h3><%= raw t("analytics.hypertension_controlled") %></h3>
     <p class="desc">
-      <%= t("analytics.hypertension_controlled_desc") %>
+      <%= raw t("analytics.hypertension_control_desc") %>
```

Comment:
This is a separate issue, we were calling the wrong locale key, which was rendering like,

Before: https://simpledotorg.slack.com/files/UFL4Z1QBV/F01HBBWD0MB/correction_3.jpeg
After: https://receive.nilenso.com/api/download/a43b19e2-44f7-11eb-ba6a-6adbae69d9d8

(github's image uploader is broken)